src/Service/Athena/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Added `us-iso-east-1` region
 - AWS api-change: You can now define custom spark properties at start of the session for use cases like cluster encryption, table formats, and general Spark tuning.
+- BC BREAK: Removing SparkProperties from EngineConfiguration object for StartSession API call
```

Comment:
We can just remove this line and the above line, since this reverts the other unreleased change.